JumpscaleLibs/clients/minio/MinioHealingClient.py

```diff
@@ -0,0 +1,112 @@
+"""
+Minio Healing Client
+"""
+
+from Jumpscale import j
+import requests
+from urllib.parse import urlencode, urljoin
+
+JSConfigClient = j.baseclasses.object_config
+
+
+class MinioHealingClient(JSConfigClient):
+    """
+    Minio healing client object
+    """
+
+    _SCHEMATEXT = """
+        @url = jumpscale.minio.client
+        name** = "" (S)
+        url** = "" (S)
+        """
+
+    def get_jobs(self):
+        resp = requests.get(f"{self.url}/jobs")
+        resp.raise_for_status()
+        jobs = resp.json()
+    
+        minio_jobs = []
+        for job_id, job in jobs.items():
+            minio_jobs.append(MinioJob(self.url, job_id=job_id, job=job))
+        return minio_jobs
+
+    def heal(self, report=None, dry_run=False, bucket=None, object_path=None):
+        getVars = {}
+
+        if report:
+            getVars["bg"] = 1
+        if dry_run:
+            getVars["dry_run"] = 1
+
+        url = urljoin(self.url, "repair/")
+        if bucket is not None:
+            url = urljoin(url, f"{bucket}/")
```

Comment:
I am sorry but actually the `/` at the end should be ONLY added if you are joining an object. otherwise if u want to heal a full bucket it should only be `{bucket}`  without the "/".